packages/smartgpt-bridge/src/indexer.ts

```
@@ -21,556 +21,581 @@ let EMBEDDING_INSTANCE = null; // cached default embedding fn
-  while (start < text.length) {
-    const end = Math.min(text.length, start + maxLen);
-    const chunk = text.slice(start, end);
-    const startLine = text.slice(0, start).split("\n").length;
-    const endLine = text.slice(0, end).split("\n").length;
-    chunks.push({ index: i++, start, end, startLine, endLine, text: chunk });
-    if (end === text.length) break;
-    start = end - overlap;
-  }
-  return chunks;
+	const chunks = [];
+	let i = 0;
```

ESLint

[error] Unexpected let, use const instead.

(functional/no-let)